package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "name-request",
-  "version": "5.0.34",
+  "version": "5.0.35",
   "lockfileVersion": 1,
```

Comment:
I don't know how I missed this, but this should be lockfile version 2.

I think you are using an old version of Node. It should be:

```
severin@DESKTOP-2XZPXD3:~$ node --version
v16.14.2
```

Reply:
Also make sure you use `npm i` to re-create the package lock file.

Reply:
Looks fixed to me :)